core/modules/css/KvInterfaceImplMySql.cc

```diff
@@ -114,66 +114,23 @@ namespace lsst {
 namespace qserv {
 namespace css {
 
-// this is copied more or less directly from QMetaTransaction. Refactor to shared location?
-// proper place is to port it into db module
-// https://github.com/LSST/db
-// right now it's python only.
-// create a ticket to make generalized code to be used between c++ and python
-class KvTransaction
-{
+
+class KvTransaction : public sql::SqlTransactionStd {
```

Comment:
This definitely collides with `KvTransaction` in other file. It can explode, be careful. One Definition Rule can do horrible things if violated.